static/PITS/post_action_json.schema.json

```diff
@@ -21,6 +21,10 @@
           "$ref": "file_input.schema.json#/definitions/input_objects_action",
           "description": "An array of action objects you wish photoshopActions to execute. We currently only support one input object."
         },
+        "costOptimization": {
```

Comment:
@johnleetran I believe we don't expose costOptimization in public docs, correct? If so, I will update this schema file for docs.